simplex.py

```diff
@@ -170,6 +170,8 @@ def run(self, r_data=None):
                 ff.score = compare.compare_data(r_data, data)
                 opt.pretty_ff_results(ff)
         self.new_ffs = sorted(self.new_ffs + [ff_copy], key=lambda x: x.score)
+        # Allow 3 cycles w/o change for each parameter present
+        self.max_cycles_wo_change = 3*(len(self.new_ffs)-1)
```

Comment:
This should not overwrite what is declared in `__init__`. This should rather be something like,

```
if not self.max_cycles_wo_change:
    self.max_cycles_wo_change = 3*(len(self.new_ffs)-1)
```

The other option is to change `self.max_cycles_wo_change` to `self._max_cycles_wo_change`, meaning that the user should not play with that setting. See this [discussion](https://stackoverflow.com/questions/6930144/underscore-vs-double-underscore-with-variables-and-methods) for explanation.